### ISSUE TITLE
Add composer.json and more useful classes 

### ISSUE THREAD

**Issue by REPTILEHAUS** [REPTILEHAUS/CHAMELEON_ONE#5]
More classes would be great, what do you use ?

It would be great to have  a composer.json file in the root that would add various classes to the "/inc/classes" folder in the theme.

Its something ive been meaning to do but haven't gotten around to yet.

**Comment by Braunson** [comment]
Are you just trying to save any packages required by composer into the /inc/classes folder and autoload them all from there as well?

**Comment by REPTILEHAUS** [comment]
Yeah thats basically the gist of it, but if you can add any improvements or other nice to haves that would be cool too.
